src/kv/kvtypes.h

```diff
@@ -14,8 +14,12 @@
 
 namespace kv
 {
-  using Index = int64_t;
+  // Version indexes modifications to the local kv store. Special value -1
+  // indicates deletion
   using Version = int64_t;
+  // Term describes an epoch of Versions. It is incremented when global kv's
+  // writer(s) changes. Term and Version combined give a unique identifier for
+  // all accepted kv modifications
```

Comment:
It might be worth noting that the kv does not actually know about the Term and that the mapping between Terms and Versions is handled by Raft via the term_history.

Reply:
Yes, the KV should not mention terms!